docs/data/data-grid/row-selection/row-selection.md

```diff
@@ -49,6 +49,13 @@ To activate checkbox selection set `checkboxSelection={true}`.
 
 {{"demo": "CheckboxSelectionGrid.js", "bg": "inline"}}
 
+### Select only visible rows [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
```

Comment:
Nitpick: A minor thing, but we tend to add the commercial items after the MIT ones for better organization ([for example](https://mui.com/x/react-data-grid/column-menu/)), so probably push this heading after the subsequent one (Custom checkbox column) to have a structure like:

- Checkbox selection
  - Custom checkbox column
  - Select only visible rows [pro]
  
We could possibly also make the tenses same:

```diff
 - Checkbox selection
   - Custom checkbox column
-  - Select only visible rows
+  - Visible rows selection
```